assets/volumedatasourcevalidator/04_deployment.yaml

```diff
@@ -0,0 +1,87 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: volume-data-source-validator
+  namespace: openshift-cluster-storage-operator
+  annotations:
+    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+    openshift.io/required-scc: restricted-v2
```

Comment:
These annotations should be in `.spec.template.metadata` instead -- one of the failing tests complains because `openshift.io/required-scc` is missing from the pod. See [aws-ebs-csi-driver-operator](https://github.com/openshift/cluster-storage-operator/blob/6f5f3f925288fdbdb7bd6ff776cd4d6c20376a9e/assets/csidriveroperators/aws-ebs/standalone/generated/apps_v1_deployment_aws-ebs-csi-driver-operator.yaml#L17-L19) for example.